app/src/main/java/org/simple/clinic/bp/BloodPressureMeasurement.kt

```diff
@@ -71,6 +71,9 @@ data class BloodPressureMeasurement (
     @Query("SELECT * FROM bloodpressuremeasurement WHERE uuid = :uuid LIMIT 1")
     fun getOne(uuid: UUID): BloodPressureMeasurement?
 
+    @Query("SELECT * FROM bloodpressuremeasurement WHERE uuid = :uuid AND deletedAt IS NOT NULL")
+    fun getOneWhichIsDeleted(uuid: UUID): Flowable<BloodPressureMeasurement>
```

Comment:
Wait, the reason we use `getOne()` is to get around the fact that we have two functions for fetching BP measurements -- one immediately and one through Rx. The Rx ones aren't called `get…` like this one.

Reply:
Do you want to avoid checking `deletedAt` in sql and do it in code instead? It'll help us avoid the confusion with the naming because I have multiple feedback on the names in this PR.

```kotlin
@Query("SELECT * FROM bloodpressuremeasurement WHERE uuid = :uuid")
fun bp(uuid: UUID): Flowable<BloodPressureMeasurement>
```

Reply:
Good idea, I'll make this change.